packages/worker/src/workflow/reusable-vm.ts

```diff
@@ -88,7 +90,8 @@ export class ReusableVMWorkflowCreator implements WorkflowCreator {
         },
       }
     );
-    const globals = { AsyncLocalStorage, assert, __webpack_module_cache__ };
+    const SafeBuffer = { ...Buffer, allocUnsafeSlow: Buffer.alloc, allocUnsafe: Buffer.alloc };
```

Comment:
Does this remove the Buffer constructor?

Reply:
Yes, Buffer constructor is supposed to be deprecated because is unsafe, do we need to keep it?

Reply:
Removed Buffer and URL changes from this PR